src/Analytics/DB/DataCollector.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace PhpAb\Analytics\DB;
```

Comment:
It's mainly because I know that this will be used for DB and cannot imagine if this will be so generic to be outside the DB namespace.